app/models/service.rb

```diff
@@ -98,7 +98,10 @@ def direct_vms
   def all_vms
     direct_vms + indirect_vms
   end
-  alias_method :vms, :all_vms
+
+  def vms
+    all_vms
+  end
```

Comment:
we want to make sure that the child implementation of `all_vms` is called.

of note, another PR is rewriting the `Service#all_vms`